presidio-image-redactor/presidio_image_redactor/image_redactor_engine.py

```diff
@@ -11,7 +15,10 @@ class ImageRedactorEngine:
     :param image_analyzer_engine: Engine which performs OCR + PII detection.
     """
 
-    def __init__(self, image_analyzer_engine: ImageAnalyzerEngine = None):
+    def __init__(
+        self,
+        image_analyzer_engine: Union[ImageAnalyzerEngine, QRImageAnalyzerEngine] = None,
```

Comment:
If `QRImageAnalyzerEngine` inherits from `ImageAnalyzerEngine`, then this class could be independent of the QR implementation